boards/arm/pinetime_devkit0/doc/index.rst

```diff
@@ -0,0 +1,25 @@
+.. pinetime_devkit0:
+
+Pine64 PineTime DevKit0
+#######################
+
+Overview
+********
+
+The PINE64 SmartWatch, dubbed "PineTime", is a product of a community effort
+for an open source smartwatch in collaboration with wearable RTOS and Linux
+app developers/communities.
+
+.. figure:: PineTime_DevKit0.jpg
+   :width: 600px
+   :align: center
+   :alt: Pine64 PineTime
+
+   Board Name (Credit: <owner>)
+
+
+References
+**********
+
```

Comment:
Normally, links to other docs like this won't be displayed in the generated HTML. If you do want them to show up, add a directive before the first link:

```
.. target-notes::

.. _Pine64 wiki: https://wiki.pine64.org/index.php/PineTime
```